domains/my.mr-julus.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Personal WebSite",
+  "owner": {
+    "username": "BougSoon-Studio",
+    "email": "bougsoon.studio@gmail.com"
+  },
+  "record": {
+    "CNAME": "my-mr-julus.web.app"
```

Comment:
Site not found, please give a preview of your website.

Reply:
The site is now avaible ;)

Reply:
This is a preview of the website :
<img width="1440" alt="Capture d’écran 2024-08-31 à 14 47 01" src="https://github.com/user-attachments/assets/03900d95-0a4f-4589-a449-ceb8000ba700">